src/journey-maps/angular/types.d.ts

```diff
@@ -0,0 +1,46 @@
+// TOOD ses: Why doesn't it work with https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/mapbox__point-geometry
+
+declare module '@mapbox/point-geometry' {
```

Comment:
@kyubisation I couldn't get it to work with the definitely typed types. (_Could not find a declaration file for module '@mapbox/point-geometry'_)

Do you have any idea or should I keep it like this for the moment?